sass/blocks/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,99 @@
+.main-nav {
+  display: block;
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 100%;
+  font-weight: bold;
+  font-size: 100%;
+  text-align: center;
+  z-index: 1;
+
+  &__list {
+    display: block;
+    position: relative;
+    width: 100%;
+    background: $color-prmiary;
+    z-index: 2;
+  }
+
+  &__item:first-child {
+    border-top: 0;
+  }
+
+  &__item {
+    display: block;
+    position: relative;
+    height: 5.6rem;
+    border-top: 1px solid $top-border;
+    border-bottom: 1px solid $bottom-border;
+    list-style-type: none;
+  }
+
+  &__item:last-child {
+    height: 5.5rem;
+    border-bottom: 0;
+  }
+
+  &__item:hover {
+    background: $color-hover-btn;
+    color: yellow;
+  }
+
+  &__item:active {
+    background: $color-active-btn;
+  }
+
+  &__anchor {
+    display: block;
+    padding-top: 1.6rem;
```

Comment:
Center with line-height and height of the parent element. Read the article on vertical centring https://css-tricks.com/centering-css-complete-guide/